### DISCUSSION TITLE
Router.push does not work.

### DISCUSSION THREAD

**Discussion by pruge** [vercel/next.js#15390] [Help] [answered]
# Bug report

## Describe the bug

/markdown 
> Entering the address directly works fine.

/ 
> Nothing happens to go using Router.push('/markdown') 
> status is always pending.

## To Reproduce

https://codesandbox.io/s/nextjs-routerpush-syvzx

**Comment by jamesmosier** [comment]
I tested this in a local project and it works as expected. Codesandbox has a tendency to have unexpected results with Next.js projects in my personal experience.

If you have a repository on GitHub that reproduces this I'd be happy to take a look.

**Comment by pruge** [comment]
https://github.com/pruge/next-route-push

When I try it still doesn't work.

**Comment by jamesmosier** [comment, marked as answer]
Seems like the issue is with `@uiw/react-md-editor`. If you remove that from your `markdown.js` page it will render correctly when doing client side navigation.

It seems that if you run `next build && next start` it works as well. So this seems to be some conflict between that package and local development in Next.

**Comment by pruge** [comment]
Thank you. I'll use @uiw/react-md-editor if it works by building.
